### PR TITLE
Replace html content in main__questions with const and render function

### PR DESCRIPTION
**Pull request by mbosselmann**
Add renderCard function which uses an array, where the data of main__questions is stored and remove content of index.html. Everything seems to work fine. Feedback is much appreciated. :-)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -57,6 +57,152 @@ buttonElHome.addEventListener('click', () => {
   buttonElHome.classList.add('item__link--position')
 })
 
+// Data
+
+const questionsObj = [
```
Comment (macfrei) [inline main.js:62]: `questionsObj` is not a good name for an array. Better would be just `questions`.

File: main.js [hunk 2]
```diff
@@ -57,6 +57,152 @@ buttonElHome.addEventListener('click', () => {
...
+      "The hand belongs to a portrait of Mrs Andrews in <strong>'Mr and Mrs Andrews'</strong> by <strong>Thomas Gainsborough</strong>. It's located in the <strong>National Gallery in London</strong> and was painted <strong>around 1750</strong>.",
+    isBookmarked: false,
+    showAnswer: false,
+    tags: ['19th century', 'painting', 'England'],
+  },
+]
+
+// Question cards
+
+function renderCard(questionsObj) {
```
Comment (macfrei) [inline main.js:117]: A better name here would be `question` as in your `renderCard` function you only get one question.

File: main.js [hunk 3]
```diff
@@ -57,6 +57,152 @@ buttonElHome.addEventListener('click', () => {
...
+
+  const cardContentQuestion = document.createElement('p')
+  cardContentQuestion.classList.add('card-content__question')
+  cardContentQuestion.textContent = questionsObj.question
+  cardContent.appendChild(cardContentQuestion)
+
+  const questionImage = document.createElement('img')
+  questionImage.classList.add('card-content__img')
+  questionImage.classList.add('img-select')
+  questionImage.setAttribute('aria-label', 'Detail of masterpiece for question')
```
Comment (macfrei) [inline main.js:149]: This should be an `alt` attribute. 🙂

File: main.js [hunk 4]
```diff
@@ -57,6 +57,152 @@ buttonElHome.addEventListener('click', () => {
...
+  answerHeader.appendChild(answerTitle)
+
+  const answerContent = document.createElement('div')
+  answerContent.classList.add('answer__content')
+  cardContentAnswer.appendChild(answerContent)
+
+  const answerImage = document.createElement('img')
+  answerImage.classList.add('content__img')
+  answerImage.classList.add('img-select')
+  answerImage.setAttribute('aria-label', 'Full view of masterpiece')
```
Comment (macfrei) [inline main.js:181]: Same here, should be an `alt` attribute.

File: main.js [hunk 5]
```diff
@@ -57,6 +57,152 @@ buttonElHome.addEventListener('click', () => {
...
+    const tagsItem = document.createElement('li')
+    tagsItem.classList.add('tags__item')
+    tagsItem.textContent = tag
+    tagsList.appendChild(tagsItem)
+  })
+}
+
+questionsObj.forEach(object => {
+  renderCard(object)
+})
```
Comment (macfrei) [inline main.js:204]:
```suggestion
questions.forEach(question => {
  renderCard(question)
})
```

**Review by macfrei — COMMENTED** [review]
Nice one! Added a few thoughts about naming 👍

File: main.js (hunk 1 above)
Comment (mbosselmann) [reply main.js]: Ok! Will change that. :-)

**Pull request merged**